scripts/models/TownHall.js

```diff
@@ -4,7 +4,7 @@
       this[key] = opts[key];
     }
   }
-  //Global data stete
+  //Global data state
```

Comment:
ha! thanks!

Reply:
No worries I make typos all the time too -_-  Half tempted to enable spellcheck.